### PR TITLE
Add procedure on adding cards

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Add procedure on adding cards

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/dashboards/cards.markdown [hunk 1]
```diff
@@ -10,9 +10,51 @@ Each dashboard is made up of cards.
...
+
+{% details "Using the Add Card button" %}
+
+1. In the bottom right, select **Add card**.
+
+2. There are two methods to add a card: **By card** type and **By entity**:
+   - **By card** type:
+        ![Add card by Card type dialog](/images/blog/2024-03-dashboard-chapter-1/sections-add-card-by-card.png)
+
+        - If you have an idea of what card tpye you want to use for an entity, browse the list of available cards.
```
Comment (frenck) [inline source/dashboards/cards.markdown]:
```suggestion
        - If you have an idea of what card type you want to use for an entity, browse the list of available cards.
```

File: source/dashboards/cards.markdown [hunk 2]
```diff
@@ -10,9 +10,51 @@ Each dashboard is made up of cards.
...
+
+{% enddetails %}
+
+{% details "Using the Add to dashboard button on device page" %}
+
+Another way to add entities to a dashboard is to use the **Device** page.
+
+1. Go to **{% my integrations title="Settings > Devices & services" %}**.
+2. On the integration card of interest, select **Devices**. 
+   - If there are multiple devices, from the list, select the device.
```
Comment (frenck) [inline source/dashboards/cards.markdown]: Starting with the condition might be helpful, but I feel like the sentence doesn't flow?

Anyways, suggestion, feel free to ignore:

```suggestion
   - If there are multiple devices, select the device from the list.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/dashboards/cards.markdown (hunk 2 above)
Comment (c0ffeeca7) [reply source/dashboards/cards.markdown]: yes... I was following the rule condition > location > action. But I get your point. Rephrased your suggestion a bit to make the syntax more natural while keeping the condition in first position.

**Review by frenck — APPROVED** [review]
Thanks, @c0ffeeca7 👍 

../Frenck

**Pull request merged**